src/Components/Publishing/ToolTip/__tests__/LinkWithTooltip.test.js

```diff
@@ -55,6 +61,9 @@ describe("LinkWithTooltip", () => {
     props = {
       url: "https://www.artsy.net/artist/nick-mauss",
       text: "Nick Mauss",
+      tracking: {
+        trackEvent: jest.fn()
+      }
```

Comment:
Perhaps a test for the hover tracking? Unless you have it somewhere that I'm not seeing.